spec/broadside/configuration_spec.rb

```diff
@@ -25,18 +25,61 @@
     expect(Broadside.config.valid?).to be false
   end
 
-  context 'ssh' do
+  describe '#ssh_cmd' do
     let(:ip) { '123.123.123.123' }
+    let(:user) { 'test-user' }
```

Comment:
you don't need to define this it's already in scope because of `include_context 'deploy configuration'`